EXE/subtitles.xml

```diff
@@ -187,43 +190,43 @@
 <Static AudioFile="echo_m04.utf|0x9B797FCD" TextArea="Cinematic">Herr Creitmire, you're being misled! Manipulated! Let go of the escort!</Static>
-<Static AudioFile="echo_m04.utf|0x89A49807" TextArea="Cinematic">Words – they take too much time. Torpedoes are faster.</Static>
+<Static AudioFile="echo_m04.utf|0x89A49807" TextArea="Cinematic">Words - they take too much time. Torpedoes are faster.</Static>
 
 <Static AudioFile="mod\m04\m04_reward_0010_hatcher.wav" TextArea="Cinematic">Gentlemen.</Static>
 <Static AudioFile="mod\m04\m04_reward_0020_trent.wav" TextArea="Cinematic">Miss Hatcher.</Static>
 <Static AudioFile="mod\m04\m04_reward_0030_alaric.wav" TextArea="Cinematic">Miss Hatcher, you have such lovely, deep eyes.</Static>
-<Static AudioFile="mod\m04\m04_reward_0040_hatcher.wav" TextArea="Cinematic">Cut this nonsense immediately, you don’t want to irritate me.</Static>
-<Static AudioFile="mod\m04\m04_reward_0045_hatcher.wav" TextArea="Cinematic">Let’s get to our business. You’ve done a fine job.</Static>
-<Static AudioFile="mod\m04\m04_reward_0050_hatcher.wav" TextArea="Cinematic">I won’t bother asking how you managed to escape Rheinland with THIS kind of documents.</Static>
-<Static AudioFile="mod\m04\m04_reward_0055_hatcher.wav" TextArea="Cinematic">I’m a delicate lady, I’m afraid I won’t be able to handle it.</Static>
+<Static AudioFile="mod\m04\m04_reward_0040_hatcher.wav" TextArea="Cinematic">Cut this nonsense immediately, you don't want to irritate me.</Static>
+<Static AudioFile="mod\m04\m04_reward_0045_hatcher.wav" TextArea="Cinematic">Let's get to our business. You've done a fine job.</Static>
+<Static AudioFile="mod\m04\m04_reward_0050_hatcher.wav" TextArea="Cinematic">I will avoid asking how you managed to escape Rheinland with THIS kind of material.</Static>
+<Static AudioFile="mod\m04\m04_reward_0055_hatcher.wav" TextArea="Cinematic">I'm a delicate lady, I'm afraid I might not be able to handle it.</Static>
 <Static AudioFile="mod\m04\m04_reward_0060_trent.wav" TextArea="Cinematic">We did what we had to do.</Static>
-<Static AudioFile="mod\m04\m04_reward_0070_hatcher.wav" TextArea="Cinematic">Heh, seems like you had fun. And by the way, we’re looking for top-notch skilled workers such as yourselves to help us perform, say, some non-standard tasks. Have you got any plans for the near future?</Static>
-<Static AudioFile="mod\m04\m04_reward_0080_alaric.wav" TextArea="Cinematic">We’ve always got time for you...</Static>
-<Static AudioFile="mod\m04\m04_reward_0090_trent.wav" TextArea="Cinematic">Shut up, Alaric. Since we’re now considered enemies of the Rheinland state, I suppose we don’t have any plans yet.</Static>
+<Static AudioFile="mod\m04\m04_reward_0070_hatcher.wav" TextArea="Cinematic">Heh, seems like you had fun. And by the way, We Are looking for top-notch skilled workers such as yourselves to help us perform, say, some non-standard tasks. Have you got any plans for the near future?</Static>
```

Comment:
I had intended to change "workers" in this line to "Freelancers" but forgot to implement the change.
```suggestion
<Static AudioFile="mod\m04\m04_reward_0070_hatcher.wav" TextArea="Cinematic">Heh, seems like you had fun. And by the way, We Are looking for top-notch skilled Freelancers such as yourselves to help us perform, say, some non-standard tasks. Have you got any plans for the near future?</Static>
```